lib/matplotlib/mpl-data/stylelib/xkcd.mplstyle

```diff
@@ -0,0 +1,37 @@
+###LINES
+
+lines.linewidth    : 2.0
+
+
+###FONT
+
+font.family        : Humor Sans, Comic Sans MS
+font.size          : 14.0
+
+##AXES
+
+axes.linewidth     : 1.5
+axes.grid          : False
+axes.unicode_minus : False
+axes.edgecolor     : black
+
+##PATH
+
+path.sketch        : (scale, length, randomness)
```

Comment:
Shouldn't these be numbers, rather than the placeholders "scale, length, randomness"?